lib/core/app/operations/cloud_events_stream.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CloudEventsStream
+  def call(type, message_id, data)
```

Comment:
This does not follow the pattern that we've agreed

Reply:
Please refer to this PR: https://github.com/rails-on-services/ros/pull/105/files

Reply:
fixed